src/tersets.zig

```diff
@@ -53,6 +54,12 @@ pub const Segment = struct {
     end_point: DiscretePoint,
 };
 
+/// Linear function of the form y = slope*x+intercept. It uses f80 for numerical stability.
```

Comment:
As mentioned in #50, maybe create a shared folder containing all helper types, functions, and methods. See [this comment](https://github.com/cmcuza/TerseTS/pull/50#discussion_r1827024730) for more details.

Reply:
That makes a lot of sense. Then, I will keep this PR open, even if all the issues are solved. Once I merge the other PR, I will ask for another review for this one.

Reply:
Or if you want to accept this PR first and later I move the Convex Hull to another folder, we can do that as well.

Reply:
In the other PR, I address this problem, let me know if you accept that idea and if I should reproduce in this one.

Reply:
I suggest you create an issue to make this change after the two PRs have been merged. Since GitHub does not differentiate between code that has been moved and code that has been changed, I find it very hard to review a PR that does both.

Reply:
Ummm I see, ok I will do that.

Reply:
Ok, I have approved the PR as  this is the last unresolved conversation and I have no more comments. Maybe post a link to the issue in this conversation when it is created and link to this conversation in the issue so the connection is clear.

Reply:
Done!

Reply:
I will resolve the conversation :)